h/assets.yaml

```diff
@@ -220,6 +220,7 @@ site_css:
   filters: cleancss
   contents:
     - h:static/styles/vendor/icomoon.css
+    - h:static/styles/vendor/css/font-awesome.css
```

Comment:
Perhaps I'm missing something, but I can't see a reason why fontawesome needs to be included in `site_css`?